RockPaperScissors.c

```diff
@@ -5,68 +5,78 @@
 
 int main(void)
 {
-    
     // Initialize random number generator
     srand(time(NULL));
-    int n;
-    int x = 0;
-    int Rock = 1; 
-    int Paper = 2; 
-    int Scissors = 3; 
+
+    // Allocate space for variables
+    int userChoice;
+    int aiChoice;
+    int round = 0;
+
+    // Initialize constants
+    const int Rock = 1; 
+    const int Paper = 2; 
+    const int Scissors = 3; 
     
     do 
     {
         printf ("Your turn! \n");
         do
         {
-            n = get_int ("Rock = 1, Paper = 2, Scissors = 3. Please enter a number to play: \n");
+            userChoice = get_int ("Rock = 1, Paper = 2, Scissors = 3. Please enter a number to play: \n");
         }
-        while (n < 1 || n > 3);
+        while (userChoice < Rock || userChoice > Scissors);
```

Comment:
It's really nice that you already have the constants above. I've replaced all the int literals with constants.

P.S. You shouldn't have [magic numbers](https://wiki.c2.com/?MagicNumber) in your code. They're [antipatterns](https://wiki.c2.com/?AntiPattern)